library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -38,7 +40,7 @@
   public EnvoyConfiguration(String statsDomain, int connectTimeoutSeconds, int dnsRefreshSeconds,
                             int dnsFailureRefreshSecondsBase, int dnsFailureRefreshSecondsMax,
                             List<EnvoyHTTPFilterFactory> httpFilterFactories, int statsFlushSeconds,
-                            String appVersion, String appId, String virtualClusters) {
+                            String appVersion, String appId, String virtualClusters, Map<String, String> nativeFilters) {
```

Comment:
Doesn't this need an order? Two lists or a list of pairs might be a better serialization.

Reply:
whoops, used to ordered map in c++. I agree, will fix.